addons/website_slides/tests/test_slide_utils.py

```diff
@@ -212,3 +212,34 @@ def test_video_urls(self):
         })
         self.assertEqual('google_drive', slide.video_source_type)
         self.assertEqual('1qU5nHVNbz_r84P_IS5kDzoCuC1h5ZAZR', slide.google_drive_id)
+
+        # test URL from Vimeo
+        slide = Slide.create({
+            'name': 'dummy',
+            'channel_id': self.channel.id,
+            'video_url': 'https://vimeo.com/545859999',
```

Comment:
Youtube example was great: list of sources, expected, then iter on the list. That way addins "a strange test case" is easy :) .